cmd/relay/events/schedulers/parallel/parallel.go

```diff
@@ -24,7 +24,8 @@ type Scheduler struct {
 	lk     sync.Mutex
 	active map[string][]*consumerTask
 
-	ident string
+	identLogKey string
```

Comment:
making this configurable feels pretty convoluted to me. can we just call it "host" or something?

Reply:
scheduler was presented as being more generic than that, not explicitly about a "host", but it is actually always used that way, so, ok

Reply:
ah, because in relay code I'm trying to move log lines about "host" to be "pds". I think "host" is too generic and if more log lines gather on "pds" it will be easier to search for problem sources in logs. I think some other users of parallel scheduler are using it to process the full relay firehose, so a different log key would make sense there.